.Rbuildignore

```diff
@@ -1,17 +1,10 @@
 debian
-inst/doc/html
-inst/doc/latex
-inst/doc/man
-inst/doc/jss.bst
-vignettes/jss.bst
 ^.*\.Rproj$
 ^\.Rproj\.user$
-vignettes/notyet
```

Comment:
Please do not randomly delete.

I am maintainer here, I had some trial files and eg this directory DOES exists here.

Can we may be simplify the scope of the PR to _just_ a rebuild of the pdf files ?

Reply:
I see `vignettes/rmd/notyet`, but not `vignettes/notyet`. This is covered by the line below `vignettes/rmd`.

Reply:
True but I still do not see what we gain from litigating this.

What problem is caused by these entries?